internal/pkg/plugin/manager.go

```diff
@@ -68,13 +71,22 @@ func GetManager() *Manager {
 	return _singleton
 }
 
+// Setter method for profile flags
+func (manager *Manager) SetFlags(profileFlag string, debugFlag bool) {
+	if (!(profileFlag == "cpu" || profileFlag == "memory" || profileFlag == "")) {
+		log.Fatal("Could not recognize inputted profile flag")
```

Comment:
can we have the message include the acceptable values? it should probably also be prefixed by `ERROR: `